src/main/config/messages_template.xml

```diff
@@ -438,409 +440,409 @@ See the accompanying LICENSE file for applicable license.
   </message>
-  
+
   <message id="DOTX045W" type="WARN">
-    <reason>The area element in an image map should specify link text for greater accessibility.</reason>
+    <reason>The &lt;area&gt; element in an image map should specify link text for better accessibility.</reason>
     <response>Link text should be specified directly when the target is not a local DITA resource.</response>
   </message>
-  
+
   <message id="DOTX046W" type="WARN">
-    <reason>Area shape should be: default, rect, circle, poly, or blank (no value). The value '%1' is not recognized.</reason>
+    <reason>Area shape should be one of: default, rect, circle, poly, or blank (no value). The value '%1' is not recognized.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX047W" type="WARN">
     <reason>Area coordinates are blank. Coordinate points for the shape need to be specified.</reason>
     <response></response>
   </message>
-    
+
   <message id="DOTX049I" type="INFO">
-    <reason>References to non-dita files will be ignored by the PDF, ODT, and RTF output transforms.</reason>
+    <reason>References to non-DITA files are ignored by the PDF, ODT, and RTF transformations.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX050W" type="WARN">
-    <reason>Default id "org.sample.help.doc" is used for Eclipse plug-in.</reason>
-    <response>If you want to use your own plug-in id, please specify it using the id attribute on your map.</response>
+    <reason>The default ID 'org.sample.help.doc' is used for the Eclipse plug-in.</reason>
+    <response>To use your own plug-in ID, specify it using the @id attribute on the map.</response>
   </message>
-  
+
   <message id="DOTX052W" type="WARN">
-    <reason>No string named '%1' was found when creating generated text; using the value '%1' in your output file.</reason>
+    <reason>No string named '%1' was found when creating generated text; using the value '%1' in the output.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX053E" type="ERROR">
-    <reason>A element that references another map indirectly includes itself, which results in an infinite loop. The original map reference is to '%1'.</reason>
-    <response></response>
+    <reason>A element that references another map indirectly includes itself, which results in an infinite loop.</reason>
+    <response>The original map reference is to '%1'.</response>
   </message>
-  
+
   <message id="DOTX054W" type="WARN">
     <reason>Conflict text style is applied on the current element based on DITAVAL flagging rules.</reason>
-    <response>Please check ditaval and dita source to make sure there is no style conflict on the element which needs to be flagged.</response>
+    <response>Check the DITAVAL and DITA source files to make sure there is no style conflict on the element that needs to be flagged.</response>
   </message>
-  
+
   <message id="DOTX055W" type="WARN">
```

Comment:
We can also deprecate this "template is deprecated" message, I don't see it in our code anymore